src/deepsparse/server/openai_server.py

```diff
@@ -175,6 +213,107 @@ async def abort_request() -> None:
             )
             return response
 
+        @app.post(
+            "/v1/completions",
+            tags=["model", "inference"],
+            response_model=CompletionResponse,
+        )
+        async def create_completion(raw_request: Request):
+            request = CompletionRequest(**await raw_request.json())
+            _LOGGER.info(f"Received completion request: {request}")
```

Comment:
nit: i am afraid that `request` can be a beefy object and printing it to terminal may not be aesthetically pleasing